spec/lib/tasks/recalculate_invalid_reasonpref_dontknow_spec.rb

```diff
@@ -2,12 +2,19 @@
 require "rake"
 
 RSpec.describe "recalculate_invalid_reasonpref_dontknow" do
+  include CollectionTimeHelper
+
   subject(:task) { Rake::Task["recalculate_invalid_rpdontknow"] }
 
   before do
     Rake.application.rake_require("tasks/recalculate_invalid_reasonpref_dontknow")
     Rake::Task.define_task(:environment)
     task.reenable
+    Timecop.freeze(previous_collection_end_date)
```

Comment:
fixing these rake tasks is tricky because they specifically only work for 2024 logs. we can use timecop to go back but these will break again next year. I think by that time we can just remove this rake task & its associated tests